CONTRIBUTION.md

```diff
@@ -0,0 +1,161 @@
+Table of Contents
+=================
+* [Code of Conduct of Eclipse Che documentation project](#code-of-conduct-of-eclipse-che-documentation-project)
+  * [Our Pledge](#our-pledge)
+  * [Our Standards](#our-standards)
+  * [Attribution](#attribution)
+* [Pull Request Process](#pull-request-process)
+   * [Release branches](#release-branches)
+* [Creating documentation](#creating-documentation)
+   * [AsciiDoc Formatting](#asciidoc-formatting)
+      * [Using test-adoc for AsciiDoc syntax verification](#using-test-adoc-for-asciidoc-syntax-verification)
+   * [Documentation Structure](#documentation-structure)
+      * [Using newdoc to create templates for new documents](#using-newdoc-to-create-templates-for-new-documents)
+   * [Writing documentation](#writing-documentation)
+      * [Style](#style)
+      * [Using vale to check the style](#using-vale-to-check-the-style)
+      * [Adding Images](#adding-images)
+   * [Building Documentation](#building-documentation)
+   * [Publishing documentation](#publishing-documentation)
+* [Getting Support](#getting-support)
+
+
+## Code of Conduct of Eclipse Che documentation project
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+### Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+
+### Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at [http://contributor-covenant.org/version/1/4][version]
```

Comment:
These new lines are very inconsistent. Are you trying to adhere to a specific column width? If so, new lines need to be applied more consistently.

Personally, I'd keep each sentence or paragraph contiguous.